src/hooks/useAppConfig.tsx

```diff
@@ -27,7 +27,7 @@ export interface CircleTile {
 
 export interface MessageTile {
   id: string;
-  providerUserIds: string;
+  providerUserIds: string[];
```

Comment:
Was this just wrong before?  Or breaking change?

Reply:
It was just wrong before 😬